helpers/UserHelper.js

```diff
@@ -0,0 +1,32 @@
+import user from '../models';
+import logger from './logger';
+
+const { User } = user;
+
+
+/**
+ * @exports UserHelper
+ * @class UserHelpers
+ * @description Contains methods to assist User
+ * */
+class UserHelper {
+  /**
+     * Find a paticular user  by emial
+     * @async
+     * @param {string} email - the user email
+     * @return {object} Returns json object
+     * @static
+     */
+  static async findByEmail(email) {
+    try {
+      const foundUser = await User.findOne({ where: { email } });
+      if (foundUser) {
+        return foundUser;
+      }
+      return null;
+    } catch (e) {
+      logger.error(e);
```

Comment:
You have done a great job on this feature @blackshady. I like the introduction of the `logger` too. While reviewing this line, I think you forgot to pass the error object `e` to the error handler using `next(e)`. This is important because, if for some technical reason, the `User.findOne()` function returns a Promise that rejects, the user at the other end will not receive a response to their request because an internal server error occurred and it was not reported to them. They are likely to get a `RequestTimedOut` response instead which may confuse them as to the root cause of the error. I would also suggest adding the `logger` line to the error handler in `index.js` so that when a server error is caught, it will be logged and an appropriate response will be returned to the user. Let me know your thoughts on this. Thanks.